server/routes.go

```diff
@@ -586,3 +587,38 @@ func streamResponse(c *gin.Context, ch chan any) {
 		return true
 	})
 }
+
+func initServer() {
+	// calling these loads the runners into a temporary directory
+	llm.GGMLRunners()
+	llm.GGUFRunners()
```

Comment:
Could be a separate PR. Setting up the runners shouldn't happen until they're needed.